.github/workflows/CI.yml

```diff
@@ -61,6 +61,21 @@ jobs:
           cd .. && git diff --exit-code
         working-directory: checker
         if: matrix.rust == 'nightly'
+      - uses: actions/checkout@v2
+        with:
+          repository: fengalin/glib-rs-two-levels-dependent
```

Comment:
I guess we'll have to move this repository into gtk-rs at some point (and rename it too)?

Reply:
OC. Let me try to fix CI, and we'll handle this.

Reply:
Sure, it was mostly a reminder for future me. :)

Reply:
How would you like to proceed? You create a fork of [my repository](https://github.com/fengalin/glib-rs-two-levels-dependent) on behalf of `gtk-rs` and I update the CI script?

Reply:
Not sure actually... Will this repository be used somewhere else? If not, we could simply create another folder in this repository and then you run it from there. Otherwise you can give me admin rights on the repository so I can migrate it here.

Reply:
> Will this repository be used somewhere else?

I don't think so.

> we could simply create another folder in this repository and then you run it from there

I didn't want to do this initially because we need the first glib dependent to use a name in `KNOWN_GLIB_EXPORTS` which is not the first of the list. That's why I used `gstreamer`. But I guess that would be fine if it's under a `test` folder.

Reply:
Then let's go for an extra folder. Maybe we should create a "glib-tests" folder and put them there instead of having multiple places...